build.gradle.kts

```
@@ -82,25 +93,17 @@ publishing {
     }
 }
 
-java {
-    withSourcesJar()
-    withJavadocJar()
-}
-
 signing {
     isRequired = isReleaseVersion && gradle.taskGraph.hasTask("publish")
-    sign(publishing.publications["mavenJava"])
-}
-
-tasks.withType<KotlinCompile> {
-    kotlinOptions.jvmTarget = "1.8"
+    // TODO updating signing
+//    sign(publishing.publications["mavenJava"])
```

This should do the trick, I think:
```suggestion
    sign(publishing.publications)
```

---

I'm not so confident, because the signing plugin is very buggy :) But it will be fixed in #4